satellite/gracefulexit/db.go

```diff
@@ -25,12 +25,12 @@ type TransferQueueItem struct {
 	Path            []byte
 	PieceNum        int32
 	DurabilityRatio float64
-	QueuedAt        time.Time
-	RequestedAt     time.Time
-	LastFailedAt    time.Time
-	LastFailedCode  int
-	FailedCount     int
-	FinishedAt      time.Time
+	QueuedAt        *time.Time
+	RequestedAt     *time.Time
```

Comment:
Just curious why we're switching to pointer now?

Reply:
sql.Rows.Scan requires pointers

Reply:
even the ints have to be pointers? just checking, I don't think I'd seen that before

Reply:
Yes, in the case that the int value is NULL in the DB, the Scan attempts to set the value to nil, which fails if it is not a pointer.